### ISSUE TITLE
Event Classification Inference

### ISSUE THREAD

**Issue by tangxiao0** [jyrao/UniSoccer#9]
Hi, I would like to verify the accuracy of the event classification inference, for example the event of a shot on goal.

So what is the input video format I need to prepare? And what is the output of the model? And how to verify the accuracy based on the output? As I am not very familiar with the process, can you elaborate on the above? Thank you very much!

**Comment by jyrao** [comment]
Thank you for following our project! I would answer them one by one:

1. As for the input video, you should better input 30s video, any format of like .mp4, .mkv, .etc would be fine, and any resolution above 224p would be fine, since the preprocess would select 30 frames averagely and resize them to 224*224. Since our model is trained on 1FPS for 30s videos, we believe 30s input would get the best performance.

2. The output of classification model is a logits of 24 classes (23 classes actually, since one meaningless class is kicked out during training). The order of these 23 classes is the same as the event classes in config files (https://github.com/jyrao/UniSoccer/tree/main/config).

3. There has already been user succeed in this task in previous github issues. I might release them soon since I didn't elaborate this code to release version so far.

Good luck with your project!

Best,
Jiayuan
